build.gradle

```diff
@@ -216,7 +216,12 @@ subprojects {
     tasks.withType(JavaCompile) {
         // Exclude generated sources from being analyzed by Error Prone.
         options.compilerArgs += [
-                '-XepExcludedPaths:.*/generated/.*'
+                // Turn Error Prone off for generated code.
+                '-XepExcludedPaths:.*/generated/.*',
```

Comment:
We should also think of putting this configuration into `config` somehow.

Reply:
Yes, I'm thinking of this as I'm doing the same under the `base` right now. Don't have good ideas at the moment. Need to dive into Gradle docs once more.